components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/ThumbnailLoader.java

```diff
@@ -241,24 +226,15 @@ public void doCall() throws Exception {
                         try {
                             store = getThumbnailStore(pxd);
                             handleBatchCall(store, pxd, userId);
-                        } catch (DSOutOfServiceException e) {
-                            // connection issue, rethrow, triggers Insight to reconnect
-                            throw e;
-                        }
-                        catch (Throwable e) {
-                            // Can't get access to thumbnail store
-                            Image thumbnail = Factory
-                                    .createDefaultThumbnail("Can't render\n[Error "+Error.NO_THUMBNAILSTORE.code+"]");
-                            currentThumbnail = new ThumbnailData(pxd.getImage()
-                                    .getId(), thumbnail, userId, true);
+                        } catch (DSAccessException e) {
+                            currentThumbnail = Factory.createThumbnailFromRes("icons/error_black_96px.png");
```

Comment:
Icon should be retrieved from the icon manager
org.openmicroscopy.shoola.env.ui.IconManager

Reply:
sorry org.openmicroscopy.shoola.util.ui I copy the wrong link